packages/adblocker-puppeteer-example/src/index.ts

```diff
@@ -5,7 +5,7 @@ import { promises as fs } from 'fs';
 
 function getUrlToLoad(): string {
   let url = 'https://www.mangareader.to/';
-  if (process.argv[process.argv.length - 1].endsWith('.ts') === false) {
+  if (process.argv[process.argv.length - 1].endsWith('.js') === false) {
```

Comment:
Is this fixing an existing bug? In the changes in this PR alone, I did not notice that files have been renamed.

Reply:
the new system compiles .ts to .js so the script had to be adopted